### DISCUSSION TITLE
No user_setup.h, can i define settings in code.ino

### DISCUSSION THREAD

**Discussion by ldijkman** [Bodmer/TFT_eSPI#1744] [Q&A - General] [answered]
Hello
Question
would not like to use the  /home/pi/arduino/libraries/tft_espi/user_setup.h, 

can i define / override user_setup.h settings in arduino code.ino ?

Greet Luberth

**Comment by Bodmer** [comment, marked as answer]
Nope, this is discussed here: #1689
